### PR TITLE
Update stanford_basic.info.yml

### PR DESCRIPTION
**Pull request by sherakama**
# READY FOR REVIEW

# Summary
- Makes this theme dependant on components module because it is.

# Review By (Date)
- Not urgent. Wait for Drupal 8.9

# Urgency
- Low, wait for Drupal 8.9

# Steps to Test

1. Check out this branch
2. Install install profile

# Affected Projects or Products
- Just this theme

# Associated Issues and/or People
- https://www.drupal.org/project/drupal/issues/474684

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)

### CODE REVIEW

File: stanford_basic.info.yml [hunk 1]
```diff
@@ -37,11 +37,5 @@ ckeditor_stylesheets:
...
-# Not working yet.
-# Waiting on: https://www.drupal.org/project/drupal/issues/474684
-# dependencies:
-#   - ui_patterns
-#   - ui_patterns_layouts
-#   - ui_patterns_library
-#   - layout_discovery
-#   - jumpstart_ui
+dependencies:
+  - components
```
Comment (pookmish) [inline stanford_basic.info.yml:41]: should this be `components:components`? I'm not sure if its the same structure as modules.

Comment (sherakama) [reply stanford_basic.info.yml:41]: Oh, I think you are correct.

**Comment by jbickar** [comment]
Per discussion, this PR does **not** need to make it into 8.4.4.

**Comment by sherakama** [comment]
This is ready to go when we get to pulling in 8.9

**Comment by sherakama** [comment]
@jenbreese @pookmish as we are now on 8.9 we can look at this PR again.

**Comment by pookmish** [comment]
probably should change the `composer.json` to require drupal/core ^8.9

**Comment by sherakama** [comment]
> probably should change the `composer.json` to require drupal/core ^8.9

Good idea. `^8.9||^9.0` work for you?

**Comment by sherakama** [comment]
@pookmish ping

File: composer.json [hunk 2]
```diff
@@ -5,5 +5,14 @@
...
-  "minimum-stability": "dev"
+  "minimum-stability": "dev",
+  "repositories": [
+    {
+      "type": "composer",
+      "url": "https://packages.drupal.org/8"
+    }
+  ],
+  "require": {
+    "drupal/core": "^8.9||^9.0"
```
Comment (pookmish) [inline composer.json:16]:
```suggestion
    "drupal/components": ">=1.0",
    "drupal/core": "^8.9 || ^9.0"
```

Comment (pookmish) [reply composer.json:16]: image

Comment (pookmish) [reply composer.json:16]: image

**Comment by pookmish** [comment]
`Uncaught PHP Exception Twig\\Error\\LoaderError: "Template "@decanter/components/logo/logo.twig" is not defined in "@basic-dist/decanter/components/global-footer/global-footer.twig" at line 18.`

**Comment by pookmish** [comment]
Does the theme also depend on jumpstart_ui module?

**Comment by sherakama** [comment]
> Does the theme also depend on jumpstart_ui module?

It did/does, but I have been trying to remove that dependency as the decanter code is in the theme as well. Thanks for catching that, I will have another pass at it.

**Comment by sherakama** [comment]
Ok, good to review again.

**Comment by sherakama** [comment]
@pookmish pls 2 review.

**Comment by pookmish** [comment]
I'm still getting the same error as https://github.com/SU-SWS/stanford_basic/pull/154#issuecomment-657670261

**Comment by sherakama** [comment]
Ah blast. That could be a pickle. 

I don't also want to declare the decanter namespace as I'd like that to sit in jumpstart-ui but I also don't want to depend on jumpstart ui.

Probably need some more twiggery...

Thanks for testing.

**Comment by joegl** [comment]
> I'm still getting the same error as [#154 (comment)](https://github.com/SU-SWS/stanford_basic/pull/154#issuecomment-657670261)

I've got this error today as well. It appears to be because the decanter templates live in dist/templates/decanter but the info.yml points to dist/templates.

**Comment by joegl** [comment]
> > I'm still getting the same error as [#154 (comment)](https://github.com/SU-SWS/stanford_basic/pull/154#issuecomment-657670261)
> 
> I've got this error today as well. ~It appears to be because the decanter templates live in dist/templates/decanter but the info.yml points to dist/templates~. I'm not exactly sure why this happens yet exactly.

**Comment by joegl** [comment]
Didn't mean to close this I clicked the wrong button.

**Pull request closed without merging**